neuro_san/client/streaming_input_processor.py

```diff
@@ -100,8 +100,7 @@ def process_once(self, state: Dict[str, Any]) -> Dict[str, Any]:
             "num_input": num_input + 1,
             "last_chat_response": last_chat_response,
             "user_input": None,
-            "sly_data": None,
-            "returned_sly_data": returned_sly_data,
+            "sly_data": sly_data,
```

Comment:
Not sure why `sly_data` would have to be reset to `None`: if the client has set something there, don't we want to keep it for next message?
And if it's actually been updated by the server, don't we want to keep that update for next message?

Reply:
This is reasonable thinking, but ...